Orders/Client/co_040121.csv

```diff
@@ -0,0 +1,6 @@
+name01#0,"""Lavernat"""
+text01#0,"""<%me>.<br>Sorry for interrupting you.<br>I am Battle Arena Administrator Lavernat."""
+text02#0,"""I'd like ARKS like you<br>to try out the Battle Arena.<br>That's the reason why I've contacted you."""
+text03#0,"""The Battle Arena is a place<br>where ARKS can compete with each other.<br>An exciting battle and a pertinent reward are waiting for you."""
+text04#0,"""If you're interested<br>please visit the Battle Lobby."""
```

Comment:
`If you're interested<br>please visit the Battle Lobby.`

`If you're interested,<br>please visit the Battle Lobby.`